specs/consensus.md

```diff
@@ -79,10 +81,10 @@ Consensus Rules
 
 ### Rewards and Penalties
 
-| name                 | type     | value | unit | description                      |
-| -------------------- | -------- | ----- | ---- | -------------------------------- |
-| `BASE_REWARD`        | `uint64` |       |      |                                  |
-| `BASE_REWARD_FACTOR` | `uint64` |       |      | Factor that scales base rewards. |
+| name                     | type     | value       | unit   | description                                             |
+| ------------------------ | -------- | ----------- | ------ | ------------------------------------------------------- |
+| `SECONDS_PER_YEAR`       | `uint64` | `31536000`  | second | Seconds per year. Omit leap seconds.                    |
+| `TARGET_ANNUAL_ISSUANCE` | `uint64` | `2 * 10**6` | `4u`   | `(= 2000000)` Target number of coins to issue per year. |
```

Comment:
Shouldn't this be a specific % of the token supply per year.

Reply:
Clarified a bit in 6ee8e10.